### DISCUSSION TITLE
Error with space in file page with "yarn dev"

### DISCUSSION THREAD

**Discussion by cjboco** [vercel/next.js#32722] [Help]
### What example does this report relate to?

with-tailwindcss

### What version of Next.js are you using?

latest

### What version of Node.js are you using?

v15.11.0

### What browser are you using?

Safari 14.0.3 

### What operating system are you using?

Mac OS Big Sure 11.2.2

### How are you deploying your application?

yarn dev

### Describe the Bug

Webpack failing with "yarn dev".

### Expected Behavior

Should not matter if space is in file path.

### To Reproduce

Create a file path with a space in it.

I just downloaded the latest "with-tailwindcss" example and when I ran the project using "yarn dev" I got this error:

```
error - ./styles/globals.css ((webpack)/css-loader/cjs.js??ref--5-oneOf-6-1!(webpack)/postcss-loader/cjs.js??ref--5-oneOf-6-2!./styles/globals.css)
Error: ENOENT: no such file or directory, stat '/Volumes/Drobo/Develop%20Folder/netflix-clone/styles/globals.css'
```

Noticing the error message states that there is not file or directory... I renamed my "Develop Folder" to "Develop", tried "yarn dev" again and the error went away. 

I've downloaded examples in the past and this is the first time this has occurred.

**Comment by jacobhq** [comment]
Maybe if you renamed your folder to `Develop` again, and then delete the `node_modules` and the `yarn.lock`. Then run `yarn install`, and see if that helps.

**Comment by cjboco** [comment]
I went ahead and tested again and the issue is no longer happening. It appears Next.js 12 fixed the issue.
(The issue was having a space in the name of a directory for the project path)
